### ISSUE TITLE
"End" + "Home" keyboard navigation does not work when items are focused

### ISSUE THREAD

**Issue by pernyl** [qlik-oss/sn-list-objects#262]
1: tab to a filterpane
2: Press enter to move into a Filterpane
3: Press "end" to move the focused item to the bottom
Nothing happens

(It works if items are not focused)
